src/Carbon/Lang/si.php

```diff
@@ -18,28 +18,28 @@
  */
 return [
     'year' => '{1}වසර 1|වසර :count',
-    'a_year' => '{1}වසරක්|වසර :count',
+    'a_year' => '{1}වසර 1|වසර :count',
```

Comment:
a_year in the singular form is supposed to be a translation for `"a year"` with no number.